nox/sessions.py

```diff
@@ -56,6 +56,16 @@ class _SessionQuit(Exception):
     pass
 
 
+class _SessionSkip(Exception):
+    pass
+
+
+class SessionStatus(object):
+    FAIL = False
+    SUCCESS = True
+    SKIP = 3
```

Comment:
These states are inconsistent within themselves. Why not just 1, 2, 3? (It seems the answer is for "backwards-compatibility with `success`")